### PR TITLE
Upgrade to v15 & latest develop

### PR DESCRIPTION
**Pull request by jamiebrynes7**
Majority of changes are in the entity templates + setting up LB on the GameLogic worker

### CODE REVIEW

File: workers/unity/Assets/BlankProject/Scripts/Workers/UnityGameLogicConnector.cs [hunk 1]
```diff
@@ -44,8 +57,16 @@ private async void Start()
...
+
             PlayerLifecycleHelper.AddServerSystems(Worker.World);
-            GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World, entityRepresentationMapping);
+            GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World, entityRepresentationMapping, gameObject);
+
+            Worker.AddLoadBalancingSystems(configuration =>
+            {
+                configuration.AddPartitionManagement(WorkerType, UnityClientConnector.WorkerType, MobileClientWorkerConnector.WorkerType);
+                configuration.AddClientLoadBalancing("Player", ComponentSets.PlayerClientSet);
+                configuration.SetSingletonLoadBalancing(WorkerType, new EntityLoadBalancingMap(ComponentSets.DefaultServerSet).AddOverride("Player", ComponentSets.PlayerServerSet));
```
Comment (paulbalaji) [inline workers/unity/Assets/BlankProject/Scripts/Workers/UnityGameLogicConnector.cs:69]: why not split out loadbalancing map into a variable like the FPS example?

Comment (jamiebrynes7) [reply workers/unity/Assets/BlankProject/Scripts/Workers/UnityGameLogicConnector.cs:69]: No reason to do it or not do it tbh

Comment (paulbalaji) [reply workers/unity/Assets/BlankProject/Scripts/Workers/UnityGameLogicConnector.cs:69]: just think it'd be nice to keep it consistent

**Review by paulbalaji — APPROVED** [review]

**Review by zeroZshadow — APPROVED** [review]
Just to check, after this change the blank project no longer supports multiple server workers right? Only if the user were to make a new worker type for the loadbalancer

**Comment by jamiebrynes7** [comment]
> Just to check, after this change the blank project no longer supports multiple server workers right? Only if the user were to make a new worker type for the loadbalancer

Correct

**Pull request merged**